interfaces/builtin/framebuffer.go

```diff
@@ -83,17 +82,6 @@ func (iface *FramebufferInterface) AppArmorConnectedPlug(spec *apparmor.Specific
 
 // Getter for the security snippet specific to the plug
 func (iface *FramebufferInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-	switch securitySystem {
```

Comment:
I think we will do a pass and enable it for everything. Now the problem is that any interface using udev tagging breaks every interface not using it that needs to access devices.